ajax/libs/planck/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "planck",
+  "filename": "planck.min.js",
+  "description": "2D JavaScript physics engine for cross-platform HTML5 game development",
+  "homepage": "http://shakiba.me/planck.js",
+  "keywords": [
+    "box2d",
+    "html5",
+    "javascript",
+    "game",
+    "physics",
+    "engine",
+    "2d",
+    "mobile"
+  ],
+  "license": "zlib",
```

Comment:
According to our [license list reference](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L338), could you please modify this to "Zlib"? Thanks.